projects/npm-tools/packages/npm-scripts/src/jsp/jspDSL.js

```diff
@@ -0,0 +1,625 @@
+/**
+ * SPDX-FileCopyrightText: © 2019 Liferay, Inc. <https://liferay.com>
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+module.exports = function (api) {
```

Comment:
I don't think `jspDSL` is the right name for this file. A DSL is a Domain-Specific Language. This file exports something like "matchers for JSP tokens". So maybe it should be called something like `getJSPTokenMatchers.js`?